investor_wallet_platform_base/security/iwp_security.xml

```diff
@@ -43,6 +43,29 @@
             </field>
         </record>
 
+        <record
+            id="res_partner_emc_user_read_structures_with_shares_rule"
```

Comment:
Maybe it's the occasion to remove the "emc" from the id ? :)

Reply:
I think we can do this during migration to v14. I don't know if the old security records are deleted if we rename them here. I could test all that, but there's so many PRs that need merging, and this repo is really only used by one client.